config/config.go

```diff
@@ -83,6 +83,39 @@ type Integrations struct {
 
 	// KubeVirt reuses a host kubevirt and makes certain CRDs from it available inside the vCluster
 	KubeVirt KubeVirt `json:"kubeVirt,omitempty"`
+
+	// ExternalSecrets reuses a host ExternalSecrets and makes certain CRDs from it available inside the vCluster
```

Comment:
Rename to "ExternalSecrets reuses a host external secrets operator and makes certain CRDs from it available inside the vCluster"